src/yb/master/master-path-handlers.cc

```diff
@@ -1543,13 +1544,15 @@ void MasterPathHandlers::HandleMasters(const Webserver::WebRequest& req,
       reg_text = Substitute("<b>$0</b>", reg_text);
     }
     string raft_role = master.has_role() ? RaftPeerPB_Role_Name(master.role()) : "N/A";
+    string uptime = UptimeString((Env::Default()->NowMicros() - master.instance_id().start_time()) / 1000000);
```

Comment:
to not hardcode this `/ 1000000`, either use something like `MonoDelta::FromMicroseconds` or `MonoTime::GetDeltaSince`